.eslintrc.js

```diff
@@ -214,6 +214,19 @@ module.exports = {
         ],
         '@typescript-eslint/prefer-ts-expect-error': 'error',
       },
+      overrides: [
+        {
+          files: [
+            'packages/puppeteer-core/src/**/*.test.ts',
```

Comment:
Not the best as we don't await the functions in the test themselves but no alternative so happy to merge.

Reply:
Yea, in theory we could change it, but mocha tests would look different from node tests.